tracer/src/Datadog.Trace/AppSec/Transports/Http/HttpTransport.Framework.cs

```diff
@@ -53,7 +59,7 @@ public IHeadersCollection GetResponseHeaders()
                 catch (PlatformNotSupportedException ex)
                 {
                     // Despite the HttpRuntime.UsingIntegratedPipeline check, we can still fail to access response headers, for example when using Sitefinity: "This operation requires IIS integrated pipeline mode"
-                    Log.Error(ex, "Unable to access response headers when creating header tags. Disabling for the rest of the application lifetime.");
+                    Log.Warning(ex, "Unable to access response headers when creating header tags. Disabling for the rest of the application lifetime.");
```

Comment:
In systems where this fails, wouldn't it be failing (and generating the trace) constantly? Maybe add a flag to write it only once?